src/App.js

```diff
@@ -1,8 +1,25 @@
+//Alberto Mancarella
+
+
+
 import React, { Component } from 'react';
 import './App.css';
 
 import Die from './Die';
 
+
+let counter = 0;
+function counter123(){
+   if(counter>11){
```

Comment:
Limits the dice roll to 12 times

Reply:
Nice Idea bro! BTW we are the same person.